x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/counter_rate.tsx

```diff
@@ -66,16 +66,25 @@ export const counterRateOperation: OperationDefinition<
   },
   getDefaultLabel: (column, indexPattern, columns) => {
     const ref = columns[column.references[0]];
-    return ofName(ref && 'sourceField' in ref ? ref.sourceField : undefined, column.timeScale);
+    return ofName(
+      ref && 'sourceField' in ref
+        ? indexPattern.getFieldByName(ref.sourceField)?.displayName
+        : undefined
```

Comment:
Why do we not require the time scale parameter here anymore? I couldn't find any weird behavior due to this, just wondering.

Reply:
you're right, I'll correct it!